staging/src/k8s.io/apiserver/pkg/endpoints/handlers/rest.go

```diff
@@ -78,7 +83,21 @@ func (scope *RequestScope) err(err error, w http.ResponseWriter, req *http.Reque
 	responsewriters.ErrorNegotiated(err, scope.Serializer, scope.Kind.GroupVersion(), w, req)
 }
 
-func (scope *RequestScope) AllowsConversion(gvk schema.GroupVersionKind, mimeType, mimeSubType string) bool {
+func (scope *RequestScope) AllowsConversion(gvk *schema.GroupVersionKind, mimeType, mimeSubType string) bool {
```

Comment:
this method is really misleading now... it's really hard to understand that gvk is nil for normal output, and only non-nil for special types

Reply:
Is that hard?  Convert nil means “no conversion” is not terribly different from calls to scheme conversion